App.tsx

```diff
@@ -5,19 +5,24 @@
 import React, { useState } from 'react';
 import { StyleSheet, TextInput, View } from 'react-native';
 
-import { CallingCodePicker } from './src';
 import { spacing } from './src/theme';
+import { ICountry } from './src/types';
+import { CallingCodePicker } from './src';
 
 const App = () => {
-  const [selectedCallingCode, setSelectedCallingCode] = useState<string>('90');
+  const [selectedCountry, setSelectedCountry] = useState<ICountry>({
+    callingCode: '90',
+    name: 'Turkey',
+    flag: '',
+  });
```

Comment:
I think this is very hard to use. We should make it simple.

Reply:
I didn't understand, will the component give all these values or user would define some? And why we need this?

Reply:
I'm asking user to give a default code to show an initial country in the picker but that seems unnecessary now, i can define a default value for it in the component